catroid/src/main/java/org/catrobat/catroid/content/bricks/LegoEv3MotorMoveBrick.java

```diff
@@ -151,8 +147,11 @@ public void onItemSelected(AdapterView<?> arg0, View arg1, int position, long ar
 			public void onNothingSelected(AdapterView<?> arg0) {
 			}
 		});
-
+		if (motorEnum == null) {
+			readResolve();
+		}
 		motorSpinner.setSelection(motorEnum.ordinal());
+		motorSpinner.setGravity(Gravity.CENTER);
```

Comment:
That was only an attempt to increase the uniformity but as I recognise now: there is no uniform way spinners are shown anyways across all bricks.
If you like I could remove this line or even all these setGravity lines from the other bricks.